pybamm/expression_tree/functions.py

```diff
@@ -61,7 +61,7 @@ def __str__(self):
         out = f"{self.name[10:-1]}("
         for child in self.children:
             out += f"{child!s}, "
-        out = out[:-2] + ")"
+        out = out[:-2] + ")"  # pragma: no cover
```

Comment:
why only the one line, should this pragma be on the entire function?

Reply:
Codacy doesn't like this _one_ line, but passes when it is excluded from cover! I will try to dig out the logs...

Reply:
Here it is: https://app.codecov.io/gh/pybamm-team/PyBaMM/commit/7bce94ed32246c1e77fd1f9eec7b2ded431410c8/blob/pybamm/expression_tree/functions.py#L70

Reply:
ha, that is a weird one! ok then lets leave it in then